app/controllers/admin/agents_controller.rb

```diff
@@ -4,22 +4,13 @@ class Admin::AgentsController < AgentAuthController
   respond_to :html, :json
 
   def index
-    agents = policy_scope(Agent)
+    @agents = policy_scope(Agent)
       .joins(:organisations).where(organisations: { id: current_organisation.id })
-      .active.order_by_last_name
-    @invited_agents = agents.invitation_not_accepted.created_by_invite.order_by_last_name
-    @complete_agents = index_params[:search].present? ? agents.search_by_text(index_params[:search]) : agents.order_by_last_name
-    @complete_agents = @complete_agents.complete
       .includes(:service, :roles, :organisations)
-      .page(params[:page])
-  end
-
-  def search
-    agents = policy_scope(Agent)
-      .joins(:organisations).where(organisations: { id: current_organisation.id })
-      .active.complete.limit(10)
-    @agents = search_params[:term].present? ? agents.search_by_text(search_params[:term]) : agents.order_by_last_name
-    skip_authorization
+      .active.order_by_last_name
```

Comment:
Je crois bien que le `order_by_last_name` est inutile ici, puisqu'à la ligne 12 on applique un ordre de tri : soit on utilise la recherche et on trie [par pertinence](https://github.com/Casecommons/pg_search#configuring-ranking-and-ordering), soit on applique `order_by_last_name`. :wink:

Reply:
Tout à fait exact. Merci